Public/Src/FrontEnd/Script/Ast/Expressions/SwitchExpressionClause.cs

```diff
@@ -16,6 +16,9 @@ namespace BuildXL.FrontEnd.Script.Expressions
     /// </summary>
     public class SwitchExpressionClause : Expression
     {
+        /// <nodoc/>
+        public bool IsDefaultFallthrough { get; set; }

```

Comment:
>public bool IsDefaultFallthrough { get; set; } [](start = 8, length = 46)

nit: you could subclass and avoid the extra field on every regular clause. It will also simplify the serialization logic below.
All in all, no big deal.

Reply:
thought about it at some point since the switch statement does that, but then I realized that it would have to expand all the visitors with another node and opted against.

---
In reply to: [274187446](https://github.com/Microsoft/BuildXL/pull/123#discussion_r274187446) [](ancestors = 274187446)